src/shared/selinux-util.c

```diff
@@ -562,6 +562,22 @@ int mac_selinux_create_file_prepare(const char *path, mode_t mode) {
 #endif
 }
 
+int mac_selinux_create_file_prepare_label(const char *path, const char *label) {
+#if HAVE_SELINUX
+        assert(path);
```

Comment:
the path argument is kinda unnecessary, it's only used for logging. Hence, I'd either drop it, or if you keep it then at least not assert on it, and allow it to be NULL